### ISSUE TITLE
Build fails.

### ISSUE THREAD

**Issue by nov** [sharplab/spring-security-webauthn#8]
At this commit, `convertToString` is renamed.
https://github.com/webauthn4j/webauthn4j/commit/f93da0ef61aab92d2621c615fa0b245ab1e3ba15#diff-177eb0f5da488a57a3608b0ac7e1e146

It causes the sample app build failure.

```bash
$ ./gradlew build
Starting a Gradle Daemon, 1 incompatible Daemon could not be reused, use --status for details

> Task :spring-security-webauthn-core:compileTestJava
/Users/uu091468/Desktop/spring-security-webauthn-master/core/src/test/java/net/sharplab/springframework/security/webauthn/converter/Base64UrlStringToAttestationObjectConverterTest.java:34: エラー: シンボルを見つけられません
        String source = new AttestationObjectConverter(cborConverter).convertToString(expected);
```

**Comment by ynojima** [comment]
Sorry for leave it failed. I acknowledged. I'll fix it today or tomorrow.

**Comment by ynojima** [comment]
Thank you for the report. Fixed in 2aacd2d39ea8166310253352f8888ed6933c4efd
